libdnf5/base/log_event.cpp

```diff
@@ -191,9 +220,16 @@ std::string LogEvent::to_string(
         case GoalProblem::INSTALLED_IN_DIFFERENT_VERSION:
             if (action == GoalAction::REINSTALL) {
                 return ret.append(utils::sformat(
-                    _("Packages for argument '{}' installed and available, but in a different version => cannot "
-                      "reinstall"),
-                    *spec));
+                    _("Installed packages for argument '{}' are not available in repositories in the same version, "
+                      "available versions: {}, cannot reinstall."),
+                    *spec,
+                    libdnf5::utils::string::join(additional_data, ",")));
+            } else if (goal_action_is_replay(action)) {
+                return ret.append(utils::sformat(
+                    _("Cannot perform {} because '{}' is installed in a different version: '{}'."),
```

Comment:
May I ask you to mark an order of arguments `{1}`. It will allow to change order for translators. In this case what about to provide a context message for translators? Subjects are not explicitly mentioned, therefore the message without a context might be not clear.

Reply:
Added the marks but I don't know what are context messages? Can we perhaps add them separately?